src/web/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ruby:2.5
+
+WORKDIR /usr/src/app
+COPY Gemfile Gemfile.lock ./
+RUN bundle install
+
+COPY . .
```

Comment:
Need copy the .config file and update the config values for PROD. ENV might be an option